crates/energy/benches/nearby.rs

```diff
@@ -149,9 +149,17 @@ fn nearby_benchmark(c: &mut Criterion) {
                     }
 
                     time.elapsed() - duration_updating_other_stuff
-                })
+                });
             },
         );
+        let mut amount_of_connections_formed = 0;
+        for connections in app.world.query::<&mut NearbyUnits>().iter(&app.world) {
+            amount_of_connections_formed += connections.units().len()
+        }
+        println!(
+            "We ended up with {} connections between {} units",
```

Comment:
Note that every connection is counted twice.

Reply:
Connections are two ways and that is subject to change but I think that is fine. OTOH we could assemble the graph as well and benchmark that and count edges if you would like.